app/cdap/components/RecipeManagement/RecipeDetails/DirectivesTable/index.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import React from 'react';
+import Table from 'components/shared/Table';
+import TableHeader from 'components/shared/Table/TableHeader';
+import TableRow from 'components/shared/Table/TableRow';
+import TableCell from 'components/shared/Table/TableCell';
+import TableBody from 'components/shared/Table/TableBody';
+import T from 'i18n-react';
+import styled, { css } from 'styled-components';
+import { grey } from '@material-ui/core/colors';
+
+const styledTableRowCSS = css`
+  border: 1px solid rgb(0 0 0 / 12%);
+  padding: 10px 16px;
+  border-bottom: none;
+`;
+
+const StyledTableHeader = styled(TableHeader)`
```

Comment:
We can keep name as just `TableHeader` as there is one header element here. Please change where-ever applicable.